webpack/scenes/ContentViews/Create/ContentViewCreateReducer.js

```diff
@@ -0,0 +1,23 @@
+import Immutable from 'seamless-immutable';
+import {
+  CREATE_CONTENT_VIEW,
+  CREATE_CONTENT_VIEW_FAILURE,
+  CREATE_CONTENT_VIEW_SUCCESS,
```

Comment:
Foreman's API middleware automatically sets the action type from the key you generated in `ContentViewsConstants.js`, `CONTENT_VIEW_CREATE`.  (see https://github.com/theforeman/foreman/blob/fb285ce995975fb255151d9385a16554b8d51295/webpack/assets/javascripts/react_app/redux/API/APIActionTypeGenerator.js#L10)  As a result, the actions that happen are not those referenced here, but rather `CONTENT_VIEW_CREATE_REQUEST` and `CONTENT_VIEW_CREATE_SUCCESS`:

![cv_create_redux_actions](https://user-images.githubusercontent.com/22042343/102123749-5074cf80-3e15-11eb-9f63-3ea7bf26564a.png)

Handling of the request-pending state also comes for free with the API middleware: https://github.com/theforeman/foreman/blob/fb285ce995975fb255151d9385a16554b8d51295/webpack/assets/javascripts/react_app/redux/API/APIReducer.js#L20

The raw API responses are also stored by the API middleware.  This reducer and constants may not be needed.

Reply:
Ya..meant to remove these..I was using this before in an earlier version so these are lingering from there.